spec/models/post_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it { should validate_presence_of(:title) }
+  it { should validate_length_of(:title).is_at_most(250) }
+  it { should validate_numericality_of(:comments_counter).only_integer.is_greater_than_or_equal_to(0) }
+  it { should validate_numericality_of(:likes_counter).only_integer.is_greater_than_or_equal_to(0) }
+
+  describe '#five_most_recent_comments' do
+    it 'returns five most recent comments for the post' do
+      post = create(:post)
+      create_list(:comment, 7, post:)
+
+      expect(post.five_most_recent_comments.length).to eq(5)
+    end
```

Comment:
- [ ] As I have suggested above and  According to the project requirement, you are meant to write unit tests for all the custom methods you created in the model. Currently, you do not have a test case for the `update_user_posts_counter` method, I suggest you add this. :+1: :100: